web/src/components/MediaControl/MediaControl.jsx

```diff
@@ -19,6 +23,17 @@ import "./MediaControl.scss";
 const MediaControl = ({ selectedSource }) => {
     // const source = status.sources[selectedSource]
     const source = useStatusStore((s) => s.status.sources[selectedSource]);
+    const meta = useStatusStore((s) => s.status.sources[selectedSource].info);
+
+    const [invertRating, setInverse] = React.useState(false); // Stores the clicked state of a song, that way it can be set to have a green thumb if the like is clicked as that metadata won't update in the backend until the song is played again
+    const [currentSong, setSong] = React.useState();
+
+    React.useEffect(() => {
+        if(currentSong != String(meta.track)){
+            setInverse(false);
+            setSong(String(meta.track));
+        }
+    }, [meta])
```

Comment:
useState on invertRating is subpar as leaving the page and reentering it, or simply entering it as a different user, will make it unclear if the song is liked or not when it was freshly liked

I do not know how to solve this as changing the internals of useStatusStore feels unwise as someone who doesn't understand it (and also it looks fragile)

Reply:
Klayton suggested looking into what the disable checkboxes do to store bool data for streams or zones, but those just update house.json once you hit "accept" or "submit"
I'm uncertain that what we want is to have the metadata file be manually rewritten by amplipi when we hit the like button, that sounds dangerous since that is managed by pianobar directly and should likely stay that way

Also I've looked into it and useStatusStore and other persistent storage methods seem to also be not quite what I want, creating a status store in App.jsx just to store temporary data for one stream seems bad too